src/frontend/src/components/common/Callout/index.tsx

```diff
@@ -1,10 +1,16 @@
 import { ReactElement } from 'react';
 import { CalloutProps } from './interfaces';
 
-const Callout = ({ text }: CalloutProps): ReactElement => {
+const Callout = ({ text, type, className }: CalloutProps): ReactElement => {
   return (
-    <section className="callout-section">
-      <div className="callout">{text}</div>
+    <section className={'callout-section' + (className ? ` ${className}` : '')}>
+      <div
+        className={
+          type === 'piaBannerStatus' ? ' callout pia-callout' : 'callout'
```

Comment:
need help for this part, it is not clear for this part

Reply:
I doubt if you need to differentiate and create a different callout styles. If you make it consistent, that should solve the problem and introduce clarity. We can connect if you need further help.

Reply:
let's connect to discuss this issue, I am create a new style because the landing page one does not fit pia design
<img width="1673" alt="Screenshot 2023-04-11 at 9 44 46 AM" src="https://user-images.githubusercontent.com/57013495/231232326-b566eb02-d9fd-4eeb-9bb1-1d5d1712848f.png">